css/index.css

```diff
@@ -1 +1,379 @@
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: square inside;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%;
+}
+
+@media (max-width: 800px) {
+	.slogan-section img {
+		display: none;
+	}
+	
+	header {
+		display: flex;
+		flex-direction: column-reverse;
+		width: 100%;
+	}
+	header .logo {
+		align-self: center;
+	}
+	header nav  {
+		display: flex;
+		justify-content: space-around;
+	
+	}
+}
+
+@media (max-width: 500px) {
+	header {
+		display: flex;
+		flex-direction: column-reverse;
+		width: 100%;
+		border-top: 1px gray solid;
+		
+
+	}
+	header nav {
+		display: flex; 
+		flex-direction: column;
+		align-items: center;
+		/*margin: 10px 0; */
+		border-top: 1px gray solid;
+		padding: 15px 0;
+		padding-top: 0;
+		margin-top: 0;
+		
+	}
+	header nav a {
+		width: 100%;
+		border-bottom: 1px gray solid;
+		align-self: center;
+		justify-items: center;
+		text-align: center;
+		padding: 1.5rem 0px;
+		
+		
+	}
+	header .logo {
+		padding: 5px 0;
+		
+	}
+	body {
+		display: flex;
+		flex-direction: column;
+	}
+	
+	body .inner-body {
+		display: flex;
+		flex-direction: column;
+	}
+}
+
+body {
+	background: rgba(5, 159, 255, .6);
+	margin: auto;
+	overflow: hidden;
+
+
+}
+.body-container {
+	max-width: 880px;
+	margin: 0 auto;
+	background: white;
+}
+.services .body-container {
+	width: 60%;
+	margin: 0 auto;
+	background: white;
+}
+header {
+	width: 100%;
+	display: flex;
+	height: auto;
+	position: relative;
```

Comment:
position relative is used when you want to shift an element out of its normal position, or you want to you position: absolute on its child element. neither apply here, so I don't think you need it.